uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/utils/ProjectFactory.java

```diff
@@ -102,12 +101,7 @@ public void create() {
 		NewFuseIntegrationProjectWizardRuntimePage secondPage = new NewFuseIntegrationProjectWizardRuntimePage(wiz);
 		if (version != null) {
 			secondPage.typeCamelVersion(version);
-		} else {
```

Comment:
Why was the else case removed? Can you explain?

Reply:
Because it is no more needed. Without this `else case` we can ensure the latest Camel version will be used for new project.

This was workaround for Camel version `2.23.2.fuse-770010-redhat-00001 (Fuse 7.7.0 GA)` which has templates available only for Spring Boot runtime, which is no more true for `Fuse 7.8` Camel version